results/cmd/api/main.go

```diff
@@ -57,7 +69,26 @@ type server struct {
 	db *sql.DB
 }
 
+// Receive TaskRun from Watcher and save it to local Sqlite Server
 func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.TaskRun, error) {
-	// TODO: implement this.
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTaskRun not implemented")
+	database := s.db
+	statement, err := database.Prepare("INSERT INTO taskrun (taskrunlog) VALUES (?)")
+	if err != nil {
+		log.Fatalf("failed to insert a new taskrun: %v", err)
+	}
+	blobData, _ := proto.Marshal(req.GetTaskRun())
+	_, err = statement.Exec(blobData)
+	if err != nil {
+		log.Fatalf("failed tp execute insertion of a new taskrun: %v", err)
+	}
+	rows, _ := database.Query("SELECT logid, taskrunlog FROM taskrun")
+	var logid int
+	var taskrunblob []byte
+	for rows.Next() {
+		rows.Scan(&logid, &taskrunblob)
+		taskrun := &pb.TaskRun{}
+		proto.Unmarshal(taskrunblob, taskrun)
+		fmt.Println(strconv.Itoa(logid) + taskrun.String())
+	}
+	return req.GetTaskRun(), nil
```

Comment:
We're not returning the row key, so this might make it harder for users to call Get since we don't enforce uniqueness on namespace/name.

This is probably fine for now, but something we should think about as we start looking at the next steps (e.g. we may decide that the current data model is not sufficient and we need to make changes).